plan/aggregation_push_down.go

```diff
@@ -260,7 +260,8 @@ func (a *aggregationOptimizer) makeNewAgg(aggFuncs []aggregation.Aggregation, gb
 		GroupByItems: expression.Column2Exprs(gbyCols),
 		groupByCols:  gbyCols,
 	}.init(a.allocator, a.ctx)
-	var newAggFuncs []aggregation.Aggregation
+
+	newAggFuncs := make([]aggregation.Aggregation, 0, len(aggFuncs)+len(gbyCols))
```

Comment:
`len(aggFuncs)+len(gbyCols)` may be extracted as a variable which can be reused by line265.

Reply:
great.